subjects/devops/easy-cloud/README.md

```diff
@@ -0,0 +1,84 @@
+
+You are a Cloud engineer and you have received a mission from a small company with limited resources and limited cloud/devops knowledge, they want to create an infrastructure with the following:
+
+- Setup WordPress on at least two servers to ensure high availability. These servers should be configured identically to serve the WordPress application.
+- Implement an auto-scaling mechanism that automatically scales the number of servers up during high load periods and scales down during low load periods. This ensures that your infrastructure can handle traffic spikes efficiently without manual intervention.
+- Configure a load balancer to distribute incoming traffic evenly across all active servers. This ensures that no single server bears too much load, improving the responsiveness of your WordPress site.
+- Utilize a network file system (NFS) for storing dynamic WordPress content, such as media uploads, plugins, and themes. This setup allows all servers to access and serve the same dynamic content, ensuring consistency across different user sessions.
+- Store static content in an object storage service, designed for high durability and availability. This storage should serve static website assets like CSS, JavaScript, and image files.
+- Integrate a Content Delivery Network (CDN) to cache and deliver static content from locations close to your users. This reduces load times and bandwidth costs while improving user experience.
+- Use a managed database service for WordPress. This database should be accessible only by your WordPress servers and not exposed to the public internet, enhancing security.
+- Ensure that the database service offers high availability features, such as automatic backups, failover mechanisms, and scalability options.
+- Design your infrastructure to be fault-tolerant by deploying servers in different physical locations or data centers, if possible. This minimizes the impact of hardware failures and network issues.
+- Implement health checks and failover strategies to automatically reroute traffic away from failed instances and maintain high availability.
+- Monitor and adjust your infrastructure according to actual needs. Use the auto-scaling feature to align operational costs with traffic patterns, ensuring you only pay for the resources you need.
+- Apply strict access controls to ensure only authorized access to your infrastructure components. This includes securing your network file system, object storage, and database.
+- Use the LTS version for WordPress application, plugins, and themes to mitigate vulnerabilities.
+- Use security groups or similar network security mechanisms to restrict inbound and outbound traffic to only necessary ports and protocols.
+- Email alerts in the event of a problem with one of the services or a skip of the budget, you can define the budget by your self.
+
+> You are not allowed to use kubernetes services or install it in the servers, the client are not familiar with it.
```

Comment:
At the end, after the comma it should be 
```
..., the client is not familiar...
```
or 
```
..., the clients are not familiar ...
```

Reply:
fixed